solana/gatekeeper-lib/src/commands/unfreeze.ts

```diff
@@ -28,11 +28,12 @@ Unfrozen
       name: "gatewayToken",
       required: true,
       description: "The gateway token to unfreeze",
-      parse: async (input: string) => new PublicKey(input),
+      // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
I think I'm just stupid and we can actually remove the async here? (and just return a Promise<>) as described.

Reply:
I guess I'm a little confused, because it seems to me like `new PublicKey(input)` isn't an async event, so why would we need a Promise here in the first place? I obviously could be incorrect, but that's how it seems on first glance.

Reply:
I set it to be `parse: (input: string): Promise<PublicKey> => Promise.resolve(new PublicKey(input))`